tests/contrib/django_util/__init__.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
```

Comment:
good point, thought it would be a good file template for Python but I suppose not